tests/roles/cinder_adoption/tasks/main.yaml

```diff
@@ -1,8 +1,54 @@
-- name: deploy podified Cinder scheduler and API
+- name: deploy podified Cinder API
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc patch openstackcontrolplane openstack --type=merge --patch '{{ cinder_scheduler_api_patch }}'
+    oc patch openstackcontrolplane openstack --type=merge --patch '{{ cinder_api_patch }}'
+
+- name: wait for Cinder API to start up
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc wait pod --for condition=Ready -l component=cinder-api
+  register: cinder_api_running
+  until: cinder_api_running is success
+  retries: 60
+  delay: 2
+
+- name: Get cinder-backup down services
+  ansible.builtin.shell: |
```

Comment:
{{ shell_header }} is missing here and at L34
i see the ci [1] failed with this "ERROR! failed at splitting arguments, either an unbalanced jinja2 block or quotes: {{ oc_header }}"

[1] https://softwarefactory-project.io/zuul/t/rdoproject.org/build/ef8a4e5832b34336b15d2c047d1f84a9